packages/cli/package.json

```diff
@@ -89,15 +91,22 @@
       "init": [
         "./src/oclif/hooks/deprecated",
         "./src/oclif/hooks/updateNotifier",
-        "./src/oclif/hooks/checkValidNodeVersion"
+        "./src/oclif/hooks/checkValidNodeVersion",
+        "./src/oclif/hooks/renderMarkdownHelp"
       ]
     },
     "topics": {
       "delete": {
-        "description": "Delete your entire integration or a specific integration version"
+        "description": "Delete your entire integration or a specific integration version."
       },
       "env": {
-        "description": "Interact with your integration's environment"
+        "description": "Interact with your integration's environment."
+      },
+      "team": {
+        "description": "Add, remove, get team members of your integration."
```

Comment:
Might change to `"Add, remove, and get team members`

Reply:
Sure, but I'd use "or" instead of "and".